pages/administratorOverview/administratorOverview.ejs

```diff
@@ -276,6 +277,7 @@
                 <%- include('courseUpdateColumn', {column_name: 'display_timezone', label: 'timezone',   course: course, i: i}) %>
                 <%- include('courseUpdateColumn', {column_name: 'path',             label: 'path',       course: course, i: i}) %>
                 <%- include('courseUpdateColumn', {column_name: 'repository',       label: 'repository', course: course, i: i}) %>
+                <%- include('courseUpdateColumn', {column_name: 'branch',           label: 'branch',     course: course, i: i,    placeholder: "Defaults to '" + config.cloneCourseDefaultBranch + "'"}) %>
```

Comment:
I like the way you implemented `placeholder` logic in these forms. However, I'd rather just treat `branch` like all the other fields and require explicit text. Could we remove all the `placeholder` logic?

Reply:
Sounds good. Done and tested. No issues.